easybuild/easyconfigs/p/p4est/p4est-2.0-gmpich-2017.08.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'p4est'
+version = '2.0'
+
+homepage = 'http://www.p4est.org'
+description = """p4est is a C library to manage a collection (a forest) of multiple
+connected adaptive quadtrees or octrees in parallel."""
+
+toolchain = {'name': 'gmpich', 'version': '2017.08'}
+toolchainopts = {'usempi': True}  
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://p4est.github.io/release/']
```

Comment:
@rubendibattista Missing SHA256 checksum (and put `source_urls` above `sources`), `--inject-checksums` will do both for you